tests/unit/test_charm.py

```diff
@@ -34,7 +34,8 @@ def parse_kyuubi_configurations(tmp_path: Path) -> dict[str, str]:
         )
 
 
-def test_start_kyuubi(kyuubi_context):
+@patch("lightkube.Client")
```

Comment:
**suggestion**: if we need to patch this everywhere, maybe we could make it a fixture with `autouse=True` in `conftest.py`

Reply:
That's a good idea! Updated.